### DISCUSSION TITLE
How to view fonts available for wezterm?

### DISCUSSION THREAD

**Discussion by santigo-dev** [wezterm/wezterm#1438] [Q&A] [answered]
Hello, I would like to know if there's a way to list the fonts that wezterm sees available to use, I have a directory with a bunch of fonts `~/.local/share/fonts/` and I use them everywhere on Plasma, the browsers, kitty and other terminals, in regards to the terminals I use nerd fonts, and I tried searching for the fonts with `fc-list` where the fonts show up, I also saw on the --help menu about `wezterm ls-list` and it only shows the preinstalled font JetBrainsMono, the problem that I see is that wezterm doesn't seem to find the fonts that I specify so each time I end up with two terminal, one telling that it couldn't find the font, even tho I'm using that same on KDE Plasma (Iosevka Nerd Font Complete) and for kitty, I tried other combinations because in the case of kitty it showed the names of the fonts with slight variations, for example instead of being 'Iosevka Nerd Font Complete' I would see 'Iosekva Nerd Font', and I saw this because `kitty list-fonts` would give an ordered list of all available fonts, does wezterm has something like this?

**Comment by wez** [comment, marked as answer]
`wezterm ls-fonts` shows which fonts wezterm will use based on your configuration.
`wezterm ls-fonts --list-system` shows that + all available fonts.

We need to amend the documentation to talk about these commands!
